fabric_plugin/tasks.py

```diff
@@ -326,6 +336,18 @@ def get_script(download_resource_func, script_path):
         return download_resource_func(script_path)
 
 
+def _hide_output(groups):
+    possible_groups = (
+        'status', 'aborts', 'warnings', 'running', 'stdout', 'stderr', 'user'
+    )
+    groups = groups or ()
+    if any(group not in possible_groups for group in groups):
+        raise NonRecoverableError(
+            '`hide_output` must be a subset of {0} (Provided: {1})'.format(
```

Comment:
This was `Hide groups must be one of`.. which is simply not true and also doesn't indicate where the problem is originating from.